provider/lxd/config.go

```diff
@@ -65,3 +73,11 @@ func (c *environConfig) validate() error {
 	// There are currently no known extra fields for LXD
 	return nil
 }
+
+func (c *environConfig) project() string {
+	project := c.attrs["project"]
```

Comment:
Is `c.attrs` guaranteed to always be non-nil?

Reply:
In non-test code, yes.